### ISSUE TITLE
Improve the UI of the website.

### ISSUE THREAD

**Issue by AnkitSharmaOnGithub** [AnkitSharmaOnGithub/covid19#2]
The website only has a basic UI structure. Kindly make the UI more attractive.

You can use any framework like Bulma, Bootstrap etc.

**Comment by arhaanb** [comment]
Hey can you assign me this task? I would like to work on it.

**Comment by AnkitSharmaOnGithub** [comment]
Yes. Go Ahead. Best of luck for Hacktoberfest. After fixing, Kindly create a PR.

**Comment by AnkitSharmaOnGithub** [comment]
Any other people who would like to work on it are also welcome.

**Comment by KhushiGangopadhyay** [comment]
hey! can i also work on this , i am new to hacktoberfest and would like to start with this

**Comment by AnkitSharmaOnGithub** [comment]
> hey! can i also work on this , i am new to hacktoberfest and would like to start with this

Hey Khushi. Kindly work on it. Best of luck for Hacktoberfest. After fixing, Kindly create a PR. I have assigned you this issue.

**Comment by KhushiGangopadhyay** [comment]
thank you so much!

**Comment by frotank** [comment]
could you assign me this ? i would like to contribute
